daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md

```diff
@@ -47,10 +55,14 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `connectionString` | Y | Input/Output | The Service Bus connection string. Required unless using Azure AD authentication. | `"Endpoint=sb://************"` |
 | `namespaceName`| N | Input/Output | Parameter to set the name of the Service Bus namespace. Required if using Azure AD authentication. | `"namespace"` |
 | `queueName` | Y | Input/Output | The Service Bus queue name. Queue names are case-insensitive and will always be forced to lowercase. | `"queuename"` |
-| `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `"60"` |
+| `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `86400` |
```

Comment:
Good idea to also change the suggested value in the YAML above. This
  # - name: ttlInSeconds # Optional
  #   value: 60. <<<

Reply:
Looks like you already did it. Thanks!